ERCS/erc-7550.md

```diff
@@ -0,0 +1,78 @@
+---
+eip: 7550
+title: Transient ERC-20 Approvals
+description: Transient ERC-20 approvals via new contract methods
```

Comment:
Since there is only one path to set a transient approval (`transientApprove`) perhaps singular "methods"?

Reply:
done